metrics/client.go

```diff
@@ -0,0 +1,25 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	RPCCallLatency = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "rpc_call_latency",
+		Help: "The duration of an RPC call in nanoseconds",
```

Comment:
Do we need that high resolution? Milliseconds are good enough for RPC calls. It's an IO operation, so nanoseconds precision might not be required ;)